redeem/gcodes/M2x.py

```diff
@@ -116,8 +112,9 @@ def execute(self, g):
     self.printer.send_message(g.prot, "Begin file list:")
     for root, directories, filenames in os.walk(list_location):
       for filename in filenames:
-        file_byte_count = os.stat(root + os.sep + filename).st_size
-        self.printer.send_message(g.prot, "{}/{} {}".format(device_id, filename, file_byte_count))
+        if not filename.startswith('.'):
```

Comment:
while a nice touch, OctoPrint tends to filter out all non-gcode files already.

I'd argue there may be cases where a user wants to see all the files present in the print folder though, even the ones that OctoPrint is filtering out.

Reply:
Well, OP doesn't filter those out. Instead it clutters the left side of my screen with entries as if I could select them to print.